agenta-cli/agenta/cli/main.py

```diff
@@ -111,10 +110,9 @@ def init(app_name: str):
             if global_backend_host:
                 backend_host = global_backend_host
             else:
-                backend_host = "http://localhost"
+                backend_host = "https://cloud.agenta.ai"
```

Comment:
why the change

Reply:
It wasn't supposed to be http://localhost in the first placed, I set it as http://localhost while testing locally and forgot to change it back before committing.